controllers/apply/awards-for-all/form.js

```diff
@@ -1348,6 +1349,10 @@ module.exports = function({
             );
         }
 
+        if (metadata && metadata.formBrand) {
+            enriched.projectName = `${metadata.formBrand}: ${enriched.projectName}`;
```

Comment:
Sounds like we might try to handle this purely through a metadata field rather so we might not need this?

Reply:
Yeah potentially, will leave it for now until it's clearer what the solution will be.